vmdk_plugin/plugin.go

```diff
@@ -281,7 +321,7 @@ func (d *vmdkDriver) Mount(r volume.MountRequest) volume.Response {
 		isReadOnly = true
 	}
 
-	mountpoint, err := d.mountVolume(r.Name, isReadOnly)
+	mountpoint, err := d.mountVolume(r.Name, status["fstype"].(string), isReadOnly)
```

Comment:
I think it's better to auto-detect the FS ( `-t auto`).  Tracking FS type in volume metadats (KV) is slightly more be error-prone.